src/styles/parts/card.scss

```diff
@@ -0,0 +1,71 @@
+.card {
+  border: 1px solid $color-gray;
+  width: 200px;
+  padding: 32px 16px 16px;
```

Comment:
Consider using variables for commonly used values like padding, margin, font-size etc. This will make your code more maintainable.